kubetest/kops.go

```diff
@@ -310,11 +377,16 @@ func (k kops) Up() error {
 	if k.args != "" {
 		createArgs = append(createArgs, strings.Split(k.args, " ")...)
 	}
-
+	if k.etcdVersion != "" {
+		overrides = append(overrides, "cluster.spec.etcdClusters[*].version="+k.etcdVersion)
+	}
+	if len(overrides) != 0 {
+		featureFlags = append(featureFlags, "SpecOverrideFlag")
+		createArgs = append(createArgs, "--override", strings.Join(overrides, ","))
```

Comment:
I see you've stuck with string.Join - is this accepted by kops?

Reply:
Need to test

Reply:
Yes this does work:

```yaml
  etcdClusters:
  - etcdMembers:
    - instanceGroup: master-us-west-1a-1
      name: "1"
    - instanceGroup: master-us-west-1a-2
      name: "2"
    - instanceGroup: master-us-west-1a-3
      name: "3"
    name: main
    version: 3.1.10
  - etcdMembers:
    - instanceGroup: master-us-west-1a-1
      name: "1"
    - instanceGroup: master-us-west-1a-2
      name: "2"
    - instanceGroup: master-us-west-1a-3
      name: "3"
    name: events
    version: 3.1.10
  nodePortAccess:
  - 34.204.201.147/32
```

Both of these where set in a cluster